### DISCUSSION TITLE
onExit is not working when click retry

### DISCUSSION THREAD

**Discussion by ognjen-it** [argoproj/argo-workflows#10044] [Q&A]
Hi all,

from 3.4.x version of Argo Workflow, onExit works only the first time. If you try to click on Retry, it will not run onExit again.
Is it a bug or you added something like onRetry? :)

**Comment by terrytangyuan** [comment]
This should be fixed in https://github.com/argoproj/argo-workflows/pull/10004.

**Reply by ognjen-it** [reply]
Thank you for replay. I will try latest image to test it :)

**Reply by ognjen-it** [reply]
@terrytangyuan 
I just tested with latest tag, but it's not working.
when I try to retry I'm getting the following error:
<img width="1095" alt="image" src="https://user-images.githubusercontent.com/44586606/205949641-9875dbbe-78e2-4137-9420-e0c6a421ff6e.png">

And after I reload page, I'm getting this:
<img width="429" alt="image" src="https://user-images.githubusercontent.com/44586606/205949342-9c04daf6-6993-4232-b003-b9232fdb876d.png">

So, that means it's not working, am I right? :)

**Reply by ognjen-it** [reply]
Have anyone any idea why it's not working?
Maybe I can write Issue?
